aicsimageio/utils/io_utils.py

```diff
@@ -49,4 +51,4 @@ def pathlike_to_fs(
             raise FileNotFoundError(f"{fs.protocol}://{path}")
 
     # Get and store details
-    return fs.open(path)
+    return fs, path
```

Comment:
What's the logic of this change? It seems sensible, Is the intent that the fs contains the stream to the data and the path is being returned for completeness, or is it a case that the path is being returned to be operated on directly in the case of a local file?

Reply:
Will leave a note about local file pointers and abstract buffered files